src/sentry/utils/samples.py

```diff
@@ -8,11 +8,20 @@
 from __future__ import absolute_import
 
 import os.path
+from datetime import datetime, timedelta
 
 from sentry.constants import DATA_ROOT
 from sentry.event_manager import EventManager
 from sentry.utils import json
 
+epoch = datetime.utcfromtimestamp(0)
+now = datetime.now()
```

Comment:
<shrug> I wanted them to be consistently apart from eachother, e.g. always 2500ms ahead of the next.